foundations/foundation-vertx/src/main/java/org/apache/servicecomb/foundation/vertx/VertxUtils.java

```diff
@@ -85,16 +86,16 @@ public static <CLIENT_POOL> DeploymentOptions createClientDeployOptions(
   }
 
   // deploy Verticle and wait for its success. do not call this method in event-loop thread
-  public static <VERTICLE extends Verticle> boolean blockDeploy(Vertx vertx,
+  public static <VERTICLE extends Verticle> Map<String, String> blockDeploy(Vertx vertx,
```

Comment:
return `Map<String, Object>` to avoid data type convert

Reply:
fixed